app/lib/features/chat/widgets/avatar_builder.dart

```diff
@@ -26,6 +27,21 @@ class AvatarBuilder extends ConsumerWidget {
         return Padding(
           padding: const EdgeInsets.only(right: 10),
           child: ActerAvatar(
+            onAvatarTap: () async {
+              final member = await ref.read(
+                memberProvider(
+                  (userId: userId, roomId: roomId),
+                ).future,
+              );
```

Comment:
We can get `member` directly in `_MemberInfoDrawer`. Is there any purpose for using it here?

Reply:
I'm thinking to make `_MemberInfoDrawer` self data driven by just passing only `opponent userId` and do rest of things by relevant providers.

All good for now. we can think on this and change it at later stage.

Reply:
at the very least we'd need `(roomId, userId)` but I explained below why I didn't go for that. If we refactored the member-providers that might be okay though.